### ISSUE TITLE
customize a loss function (e.g., vgg16) 

### ISSUE THREAD

**Issue by henry090** [henry090/fastai#131]
dice loss & feature loss, but a general way to customize the loss function will be better. https://github.com/fastai/fastai/blob/master/dev_nbs/course/lesson7-superres.ipynb https://kaggle.com/bigironsphere/loss-function-library-keras-pytorch
